### PR TITLE
Add username, created on and modified on to submission details page

### PR DESCRIPTION
**Pull request by msmner**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/420

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx [hunk 1]
```diff
@@ -83,6 +84,37 @@ const SubmissionDetails = () => {
...
+            if (!canAccessAdministration || isNil(currentSubmission)) {
+                return null;
+            }
+
+            return (
+                <div className={styles.submissionInfo}>
+                    <p className={styles.submissionInfoParagraph}>
+                        Created on:
+                        {' '}
+                        {preciseFormatDate(currentSubmission.createdOn)}
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx:98]: Destructure.

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx [hunk 2]
```diff
@@ -83,6 +84,37 @@ const SubmissionDetails = () => {
...
+                    </p>
+                    <p className={styles.submissionInfoParagraph}>
+                        Modified on:
+                        {' '}
+                        {isNil(currentSubmission.modifiedOn)
+                            ? 'never'
+                            : preciseFormatDate(currentSubmission?.modifiedOn)}
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx:105]: Destructure

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx [hunk 3]
```diff
@@ -83,6 +84,37 @@ const SubmissionDetails = () => {
...
+                    </p>
+                    <p className={styles.submissionInfoParagraph}>
+                        Username:
+                        {' '}
+                        {currentSubmission.user.userName}
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx:110]: Destructure

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts [hunk 4]
```diff
@@ -15,6 +16,11 @@ const formatDate = (
     ? moment(date).format(formatString)
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts:16]: Use `preciseFormatDate` here as it is the same.

**Review by Minkov — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts (hunk 4 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts:16]: Done.

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx (hunk 1 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx:98]: Done.

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx (hunk 2 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx:105]: Done.

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx (hunk 3 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx:110]: Done.

**Review by Minkov — APPROVED** [review]

**Review by PetarNeshkov — APPROVED** [review]

**Pull request merged**